packages/cli/src/utils.ts

```diff
@@ -177,3 +178,72 @@ export const checkIfSubDir = (parent: string, dir: string) => {
 
   return !relativePath.startsWith('..') && !isAbsolute(relativePath)
 }
+
+export const getInstallCommand = (packageManager: PackageManager) => {
+  return `${packageManager} install`
+}
+
+type GetNPMCommandArgs =
+  | {
+      commandName: string
+      packageManager: PackageManager
+      structure: 'polyrepo'
+    }
+  | {
+      commandName: string
+      packageManager: PackageManager
+      structure: 'monorepo'
+      packageName: string
+    }
+
+export const getNPMCommand = (args: GetNPMCommandArgs) => {
```

Comment:
I have no idea how to name this function 🥲

Reply:
I just got an email notification about your comment so I am replying: What about getCommandByPackageManager ?

Reply:
That's better!